swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/OrcaService.kt

```diff
@@ -73,10 +73,15 @@ class OrchestrationRequest(
   val job: List<OrcaJob>
 )
 
+// todo emjburns: service account should be configurable
```

Comment:
The user specified here isn't really the `service account` (although that must be specified somewhere else under the covers).

I don't think there's harm in this but the UI should really avoid `User: null` when it has authenticated details it could just show instead.

Reply:
We specify it here: https://github.com/spinnaker/swabbie/blob/master/swabbie-retrofit/src/main/kotlin/com/netflix/spinnaker/swabbie/retrofit/SwabbieRetrofitConfiguration.kt#L51. Service account is probably the wrong phrase.

The ui shows this when the user passed (in this case there wasn't one) doesn't match the actual credentials. Adding it into the payload lets the ui just show the "email". Do you think there's harm in doing this/merging this PR?

Reply:
cc @alanmquach

Reply:
> I don't think there's harm in this but the UI should really avoid User: null when it has authenticated details it could just show instead.
I can do this.

Reply:
Done: https://github.com/spinnaker/deck/pull/7305